docs/admin/outage-comms.rst

```diff
@@ -0,0 +1,69 @@
+#######################
+Communicating an Outage
+#######################
+
+Intended audience: Anyone who is administering a Rubin Science Platform environment, or covers for an administrator.
+
+Putting up a banner
+===================
+
+The banners that appear on the RSP home page (:doc:`squareone </applications/squareone/index>`) are provided by the :doc:`semaphore </applications/semaphore/index>` service.
+
+The payload for the content of those banners is in the repo `rsp_broadcast <https://github.com/lsst-sqre/rsp_broadcast/>`_. You will need push access to that repository.
```

Comment:
```suggestion
The content of those banners is sourced from the https://github.com/lsst-sqre/rsp_broadcast repository.
You will need push access to that repository.
```

"The payload of the content" is a weird turn of phrase, hence this suggestion.

The theme handles and displays full URLs to GitHub repos well.